### DISCUSSION TITLE
How to http://192.168.2.221/cm?cmnd=Rule ???

### DISCUSSION THREAD

**Discussion by afgcons** [arendst/Tasmota#16302] [Q&A] [answered]
I googled and found some examples, using the web commands and the switches' console I have successfully configured my Smart Switches.

Now I need to define a rule which sends the 'power 0' command 300 seconds after receiving the web command to trigger that rule. I read the [Rules documentation](https://tasmota.github.io/docs/Rules/#rule-command) but how do I enter that with web commands (http://192.168.2.221/cm?cmnd=...) or the console of the Smart Switch? I googled but found no examples to:

- define the rule

- trigger/execute the rule

Can anyone point the finger at a "How to ..." web page or provide an example of the web and console command to define and execute the rule for my Tasmota pre-flashed Smart Switch (version 10.1.0 / 2021-12-20T16:00:46), please.

**Comment by barbudor** [comment, marked as answer]
You are mixing-up the 'rule' command which is to create the rule and the trigger of the rule
To trigger a rule on request on should use an "Event"

To create the rule use these commands in the web console:
```haskel
Rule1 on event#delaypow do RuleTimer1 300 endon on Rules#Timer=1 do Power 0 endon
Rule1 1
```
You can test the rule by typing in the console:
```haskel
Event delaypow
```

You can trigger the rule from http using:
```
http://192.168.2.221/cm?cmnd=event%20delaypow
```

**Reply by afgcons** [reply]
Thanks !! 
Now it works. I wished there were more "cookbook"-type examples to help new users with no Tasmota background to get started. I am impressed, kudos to Theo Arends and contributors!
